src/index.js

```diff
@@ -12,7 +12,20 @@ function outputResult(object, options) {
   });
 }
 
+function sortObj(obj, compare) {
+  const sorted = compare ? Object.keys(obj).sort(compare) : Object.keys(obj);
+  return sorted.reduce(function (result, key) {
+    // eslint-disable-next-line no-param-reassign
+    result[key] = obj[key];
+    return result;
+  }, {});
+}
+
 function mergeEndpointsWithBase(swaggerBase = {}, endpoints = []) {
+  // To ensure consistent sorting of HTTP methods let's universally enforce the following order.
+  const methodPriority = new Map();
+  ['get', 'post', 'put', 'patch', 'delete', 'options', 'head', 'trace'].forEach((x, i) => methodPriority.set(x, i));
```

Comment:
Since logically when reading an API definition having `delete` come before a `get` operation doesn't really make sense I'm forcing the sort to be this specific order.